packages/react/src/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -248,9 +248,14 @@ Object {
     "GA": [Function],
     "GA4": [Function],
     "GTM": [Function],
+    "GoogleConsentMode": [Function],
```

Comment:
Is this supposed to be available to users? From what I can understand, this is only used by some integrations internally.

Reply:
Users can need to interact with consent mode, for example to define which consents would be mapped to their categories or disable GoogleConsentMode module itself.